openwisp_utils/commitizen/openwisp.py

```
@@ -0,0 +1,181 @@
+                "Expected format:\n\n"
+                "  [prefix] Capitalized title #<issue>\n\n"
+                "  <long-description>\n\n"
+                "  Fixes #<issue>\n\n"
+                "Example:\n\n"
+                "  [feature] Add subnet import support #104\n\n"
+                "  Add support for importing multiple subnets from a CSV file.\n\n"
+                "  Fixes #104"
+            ],
+        )
+
+    def format_error_message(self, message: str) -> str:
```

Ruff (0.14.14)

[warning] Unused method argument: `message`

(ARG002)